tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
the root is referencing its children? thought it only went the other way around?

Reply:
looks like it still builds when I remove this.

Reply:
removed